File: logicaProgramacaoPROA/visualG-and-JS/FIT-FACCAT/VISUALG/JS/exercicio38/js/exercicio38.js
let codigo, senha;

codigo=parseInt(prompt("Insira o código de entrada: "));
senha=parseInt(prompt("Insira senha: "));

switch (codigo) {
    case 1234: alert("Acesso permitido!");
    break;
    default: 
    alert("Usuário incorreto.");
}

switch (senha){
    case 9999: alert ("Senha correta!");
    break;
default: 
    alert("Senha incorreta.");

}

sim

// Escreva ("Insira o código de entrada: ")
//    Leia (codigo)
//    Escolha codigo
//    Caso 1234
//       Escreva ("Insira a senha: ")
//       Leia (senha)
//       Escolha senha
//       Caso 9999
//          Escreval ("--------------------------------------")
//          Escreval ("Acesso permitido!")
//       Outrocaso
//          Escreval ("Senha incorreta!")
//       Fimescolha
//    Outrocaso
//       Escreval ("Usuário inválido!")
//    Fimescolha

File: logicaProgramacaoPROA/JAVA/FACCAT/Exercicio38.java
package javaBasico;

import java.util.Scanner;

public class Exercicio38 {
	
	public static void main(String args[]) {
		
		//VARIÁVEIS
		int codigo, senha;
		
		// SCANNER
		Scanner sc = new Scanner(System.in);
		
		// SITEMA DE LOGIN
		System.out.println("Tela de login\n"
				+ "Insira o código para entrar: ");
		codigo = sc.nextInt();
		
		// VALIDAÇÃO DE LOGIN
		if (codigo != 1234) {
			System.out.println("Usuário inválido!");
		}
		
		// VALIDAÇÃO DE SENHA
		else {
			System.out.println("Agora, insira a senha: ");
			senha = sc.nextInt();
			if (senha != 9999) {
				System.out.println("Senha incorreta!");
			}
			else {
				System.out.println("Acesso permitido!");
			}
		}
		sc.close();
		
		// FIM DO PROGRAMA
		System.out.println("Fim do programa.");
		
	}
	
}
